### PR TITLE
added content + translation to submission review page

### PR DESCRIPTION
**Pull request by blai0264**

### CODE REVIEW

File: src/pages/submission/submission.scss [hunk 1]
```diff
@@ -67,4 +67,8 @@ page-submission {
             margin: 0 auto;
             text-justify: left;
         }
+
+        p{
+            line-height: 1.61;
```
Comment (CalvinRodo) [inline src/pages/submission/submission.scss]: Does this page have a unique line-height requirement? Should this be a global change?

Comment (blai0264) [reply src/pages/submission/submission.scss]: It does, some pages have different line-heights specified in Zeplin. For the questions on the submission review page the line-height is specified at 1.61 so I added it.

Comment (blai0264) [reply src/pages/submission/submission.scss]: Also noticed different pages have different line height so wasn't sure if it should be global or not. What do you think Calvin?

Comment (CalvinRodo) [reply src/pages/submission/submission.scss]: Anything that is unique to a page should only be in that pages css files, I think this is fine the way it is.

**Pull request merged**